src/redux/actions/experiments/updateExperiment.js

```diff
@@ -0,0 +1,21 @@
+import saveExperiment from './saveExperiment';
+import {
+  EXPERIMENTS_UPDATE,
+} from '../../actionTypes/experiments';
+
+const updateExperiment = (
+  experimentId,
+  experiment,
+) => async (dispatch) => {
+  dispatch({
+    type: EXPERIMENTS_UPDATE,
+    payload: {
+      experimentId,
+      experiment,
+    },
+  });
+
+  dispatch(saveExperiment(experimentId));
+};
```

Comment:
add dispatch for error and error state

Reply:
added

Reply:
I don't see this dispatch for error, maybe you didn't commit the changes?